markdown/kubernetes_operator_codelab/kubernetes_operator_codelab.md

```diff
@@ -0,0 +1,243 @@
+metadata:
+  name: dev-example
+spec:
+  developer: true" > developer.yaml
+
+# Then apply it
+kubectl apply -f developer.yaml
+
+# Wait for broker deployment pods to be ready
+kubectl get pods --show-labels --watch
+
+# Check service-ready
+kubectl wait --for=condition=ServiceReady eventbroker dev-example
+```
+
+![Broker Setup Verification](img/Operator_Img4.png)
+
+### Next Step is to test the deployment
+
+## Test Solace Pusbub+ Event Broker Deployment
+
+Duration: 0:05:00
+
+In this section we are going to check the event broker deployment status, get information about the service name and type to access the broker services, and obtain the secret that contains the credentials to be used for admin access.
+
+### 1. Check the Event Broker Status
+```bash
+kubectl describe eventbroker dev-example
+```
+### 2. Obtain the management admin password
+```bash
+ADMIN_SECRET_NAME=$(kubectl get eventbroker dev-example -o jsonpath='{.status.broker.adminCredentialsSecret}')
+
+# This command returns the management "admin" user's password
+kubectl get secret $ADMIN_SECRET_NAME -o jsonpath='{.data.username_admin_password}' | base64 -d
+```
+
+### 3. Obtain the IP address to access the broker services
+
+```bash
+BROKER_SERVICE_NAME=$(kubectl get eventbroker dev-example -o jsonpath='{.status.broker.serviceName}')
+
+# This command returns the broker service's external IP address
+kubectl get svc $BROKER_SERVICE_NAME -o jsonpath='{.status.loadBalancer.ingress}'
+```
+
+### 4. Access the PubSub+ Broker Manager
+
+
+* In your browser, navigate to the IP address you obtained, using port 8080
+
+```bash
+http://<ip-address>:8080
+```
+Note: If you did not get the external ip address of broker service in step 3 that means there is no external load balancer created. As a workaround, you can run port forwarding in the background.
+
+```bash
+# Run port forwarding for SEMP
+kubectl port-forward svc/$BROKER_SERVICE_NAME 8080:8080
+
+# Now you can access to localhost at port 8080 and login to Pubsub+ Manager
+http://localhost:8080
+```
+
+* Login as user `admin` with the management admin password you obtained.
+* Use the Broker Manager [built-in Try-Me](https://docs.solace.com/Admin/Broker-Manager/PubSub-Manager-Overview.htm?Highlight=manager#Test-Messages) tool to test messaging.
+
+![Pubsub+Manager Login1](img/Operator_Img5.png)
+
+![Pubsub+Manager Login2](img/Operator_Img6.png)
+
+
+## Takeaways
+Duration: 0:01:00
+
+
+This codelab shows how the operator makes an event broker deployment easy by simply applying the yaml file. We have deployed a dev instance of the Pubsub+ Event Broker using the Operator in a Kubernetes cluster, where, Kind is used to run Kubernetes locally on a mac machine. \
+Checkout [Kubernetes Operator - Quickstart](https://github.com/SolaceProducts/pubsubplus-kubernetes-quickstart) for more deployment options.
+<!--✅ < Fill IN TAKEAWAY 1>   
+✅ < Fill IN TAKEAWAY 2>   
+✅ < Fill IN TAKEAWAY 3>   -->
+
+![Soly Image Caption](img/soly.gif) 
```

Comment:
In the preview link, looks like this image is broken hmm. Did you run the `export.sh` script and made sure all the files are added and committed? 
https://644fe87763f0753cfd819359--codelabs-solace.netlify.app/codelabs/kubernetes_operator_codelab/index.html?index=..%2F..index#6

<img width="1007" alt="Screen Shot 2023-05-05 at 6 46 10 PM" src="https://user-images.githubusercontent.com/6826053/236585644-5cb552c8-11f0-405f-a63d-f2a964be98c7.png">